libraries/AP_Arming/AP_Arming.cpp

```diff
@@ -187,9 +204,17 @@ bool AP_Arming::airspeed_checks(bool report)
             return true;
         }
         for (uint8_t i=0; i<AIRSPEED_MAX_SENSORS; i++) {
-            if (airspeed->enabled(i) && airspeed->use(i) && !airspeed->healthy(i)) {
-                check_failed(ARMING_CHECK_AIRSPEED, report, "Airspeed %d not healthy", i + 1);
-                return false;
+            if (airspeed->enabled(i)) {
+                if (!airspeed->healthy(i)) {
+                    check_failed(ARMING_CHECK_AIRSPEED, report, "Airspeed %d not healthy", i + 1);
+                    return false;
+                } else if (airspeed->get_airspeed() > 5.01f) {
```

Comment:
the plane if normally pointed into the wind. This check means you can't arm and takeoff in wind over 10knots

Reply:
Changed this to over 7m/s

Reply:
That's not high enough. You have to make this an opt in parameter, or a parameter with a much higher value.

Reply:
Done